config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -944,42 +927,36 @@ def generate_network_plugins():
 ################################
 def generate_upgrades():
 
-    kops24 = 'v1.24.5'
     kops25 = 'v1.25.4'
     kops26 = 'v1.26.4'
-    kops27 = 'v1.26.7-beta.3'
+    kops27 = 'v1.27.0-beta.3'
 
     versions_list = [
         #  kops    k8s          kops      k8s
-        # 1.25 release branch
-        ((kops24, 'v1.19.16'), ('1.25', 'v1.20.6')),
-        ((kops24, 'v1.23.1'), ('1.25', 'v1.23.1')),
-        ((kops25, 'v1.21.14'), ('1.25', 'v1.22.13')),
-        ((kops25, 'v1.25.0'), ('1.25', 'v1.25.0')),
         # 1.26 release branch
-        ((kops24, 'v1.20.6'), ('1.26', 'v1.21.7')),
-        ((kops24, 'v1.24.0'), ('1.26', 'v1.25.0')),
         ((kops25, 'v1.20.6'), ('1.26', 'v1.21.7')),
         ((kops25, 'v1.25.0'), ('1.26', 'v1.26.0')),
         ((kops26, 'v1.21.14'), ('1.26', 'v1.22.13')),
         ((kops26, 'v1.26.0'), ('1.26', 'v1.26.0')),
         # 1.27 release branch
-        ((kops25, 'v1.20.15'), ('1.27', 'v1.21.14')),
+        ((kops25, 'v1.20.15'), ('1.27', 'v1.22.17')),
+        ((kops25, 'v1.21.14'), ('1.27', 'v1.22.17')),
+        ((kops25, 'v1.22.17'), ('1.27', 'v1.23.17')),
+        ((kops25, 'v1.23.17'), ('1.27', 'v1.24.15')),
+        ((kops25, 'v1.24.15'), ('1.27', 'v1.25.11')),
         ((kops25, 'v1.25.11'), ('1.27', 'v1.26.6')),
+        ((kops25, 'v1.26.6'), ('1.27', 'v1.27.3')),
         ((kops26, 'v1.21.14'), ('1.27', 'v1.22.17')),
-        ((kops26, 'v1.26.6'), ('1.27', 'v1.26.6')),
-        ((kops27, 'v1.22.17'), ('1.27', 'v1.23.17')),
-        ((kops27, 'v1.27.3'), ('1.27', 'v1.27.3')),
-        # 1.24 upgrade to latest
-        ((kops24, 'v1.23.1'), ('latest', 'v1.24.0')),
-        ((kops24, 'v1.24.0'), ('latest', 'v1.25.0')),
-        # 1.25 upgrade to latest
-        ((kops25, 'v1.24.0'), ('latest', 'v1.25.0')),
-        ((kops25, 'v1.25.0'), ('latest', 'v1.26.0')),
+        ((kops26, 'v1.22.17'), ('1.27', 'v1.23.17')),
+        ((kops26, 'v1.23.17'), ('1.27', 'v1.24.15')),
+        ((kops26, 'v1.24.15'), ('1.27', 'v1.25.11')),
+        ((kops26, 'v1.25.11'), ('1.27', 'v1.26.6')),
```

Comment:
I don't think it's worth testing all these k8s versions in the middle.

Reply:
Removed